src/components/code-editor-wrapper/index.tsx

```diff
@@ -5,6 +5,7 @@ import Moveable from 'react-moveable'
 import { OnResizeStart, OnResize } from 'react-moveable/declaration'
 
 import CodeEditor from './code-editor'
+import PreviewNextPosition from './preview-next-position'
```

Comment:
next-position-preview is the correct form

Reply:
oh sure, i will change it.